.github/workflows/host.yml

```
@@ -142,11 +142,11 @@ jobs:
           rm -rf  /config
       - name: Send Notification to Rocket Chat
         uses: RocketChat/Rocket.Chat.GitHub.Action.Notification@1.1.1
-        if: always()
+        if: always() && (contains(inputs.host_name, 'ref')) || (contains(inputs.host_name, 'prod'))
```

Does the `always()` still serve any purpose? If not, I would suggest simplifying this if statement for readability to
```suggestion
        if: contains(inputs.host_name, 'ref') || contains(inputs.host_name, 'prod')
```
And was it possible to quickly test if the `contains()` expression works as expected? For example, by deploying to `infra` and `dev` while having the if statement
```
        if: contains(inputs.host_name, 'infra')
```
and checking afterwards if only a message for the `infra` environment was sent to the test rocket chat channel.